spec/orbital/mit_cit_params_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'spec_helper'
+
+ActiveMerchant::Billing::Base.mode = :test
+
+describe Killbill::Orbital::PaymentPlugin do
+
+  include ::Killbill::Plugin::ActiveMerchant::RSpec
+
+  before(:each) do
+    @plugin = build_plugin(::Killbill::Orbital::PaymentPlugin, 'orbital')
+    @plugin.start_plugin
+    @call_context = build_call_context
+
+    ::Killbill::Orbital::OrbitalPaymentMethod.delete_all
+    ::Killbill::Orbital::OrbitalResponse.delete_all
+    ::Killbill::Orbital::OrbitalTransaction.delete_all
+  end
+
+  after(:each) do
+    @plugin.stop_plugin
+  end
+
+  context 'mit cit params spec' do
+    before(:each) do
+      @amount     = BigDecimal.new('100')
+      @currency   = 'USD'
+    end
+
+    it 'should set correct mit cit params for the first transaction with credential_on_file true' do
+      @properties = build_pm_properties(nil, { :cc_number => '5454545454545454', :cc_type => 'master' })
```

Comment:
Not sure if it matters here but this is a MasterCard - could be a bit confusing.